GWAS/LMM.ipynb

```diff
@@ -929,7 +929,7 @@
     "bash: container=container_lmm, expand= \"${ }\", stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout' \n",
     "    plink2 \\\n",
     "      --bfile ${bfile:n} --mac 1 \\\n",
-    "      ${('--maf %s' % maf_filter) if maf_filter >= 0 else ''} ${('--geno %s' % geno_filter) if geno_filter >= 0 else ''} ${('--hwe %s' % hwe_filter) if hwe_filter >= 0 else ''} ${('--mind %s' % mind_filter) if mind_filter >= 0 else ''} \\\n",
+    "      ${('--maf %s' % maf_filter) if maf_filter > 0 else ''} ${('--geno %s' % geno_filter) if geno_filter > 0 else ''} ${('--hwe %s' % hwe_filter) if hwe_filter > 0 else ''} ${('--mind %s' % mind_filter) if mind_filter > 0 else ''} \\\n",
```

Comment:
@dianacornejo this chunk of code should also be reverted?

Reply:
@gaow this chunk didn't have the `=` in the previous versions. I had added thinking this was the GWAS_QC notebook but I changed it back as how it was before (without the `=` signs)

Reply:
I see. Should we then add back the `=` for this?

Reply:
@gaow I didn't add this before because regenie will throw an error if a low-variance SNP is included in the step 1 run. Therefore this cannot be 0, you have to filter somehow. Does this make sense?

Reply:
should we call it regenie_qc if it is specific to regenie and is not used in other steps?

Reply:
that's probably a good idea.